vowpalwabbit/core/src/parse_slates_example_json.cc

```diff
@@ -2,324 +2,6 @@
 
 #include "vw/common/future_compat.h"
 
```

Comment:
explicit template initialization needs to stick around here and in `parse_example_json.cc` until the functions are deprecated